libs/browser/settings/feature/shell/src/lib/browser-settings.spec.tsx

```diff
@@ -3,7 +3,7 @@ import { render } from '@testing-library/react';
 import BrowserSettings from './browser-settings';
 
 describe('BrowserSettings', () => {
-  it('should render successfully', () => {
+  xit('should render successfully', () => {
```

Comment:
what is different between `xit` and `it`

Reply:
because I want to exclude this test case for the time being.